open_bounties/0062 Account-based toml Lookup.md

```diff
@@ -0,0 +1,44 @@
+---
+ID: "0062"
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/0062
+Category: XRPL.org tools
+Status: Open
+Amount: $2,500
+---
+
+
+# Proposal Title
+
+Account-based toml Lookup Tools
+
+**Category:**
+XRPL.org tools
+
+
+**Amount:** 
+$2,500
+
+## Overview
+
+The xrp-ledger.toml checker on XRPL.org (https://xrpl.org/xrp-ledger-toml-checker.html) is a handy tool for looking up what accounts are claimed by a specific domain (including domain verification), but it could be convenient to be able to do it the other way around and check to see if an account really is claimed by the domain it has set in its Domain field.
```

Comment:
```suggestion
The [xrp-ledger.toml checker on XRPL.org](https://xrpl.org/xrp-ledger-toml-checker.html) is a handy tool for looking up what accounts are claimed by a specific domain ([including domain verification](https://xrpl.org/xrp-ledger-toml.html#domain-verification)), but it could be convenient to be able to do it the other way around and check to see if an account really is claimed by the domain it has set in its Domain field.
```

Reply:
(Adding links in-line for more context to the reader)